samples/highcharts/accessibility/internationalization/demo.html

```diff
@@ -0,0 +1,27 @@
+<script src="https://code.highcharts.com/stock/highstock.js"></script>
+<script src="https://code.highcharts.com/stock/modules/data.js"></script>
+<script src="https://code.highcharts.com/modules/accessibility.js"></script>
+<link rel="stylesheet" type="text/css" href="//github.com/downloads/lafeber/world-flags-sprite/flags32.css" />
+
+<div role="radiogroup" style="min-width: 310px; max-width: 600px; margin: 0 auto">
```

Comment:
Recommend setting the styling in CSS, and the max width could perhaps be larger?